### DISCUSSION TITLE
Controller Path Wrong (issue or not)

### DISCUSSION THREAD

**Discussion by bhargavraviya** [adonisjs/core#4275] [Ideas]
- In Laravel controller path is app/Http/Controllers/Controller.php 
but the actual case is in adonis js app/Controllers/Http/Controller.ts
- There is an issue of path confusion or not?

**Comment by RomainLanz** [comment]
Hey @bhargavraviya! 👋🏻 

I didn't understand your issue.
Nothing is wrong.
